README.md

```diff
@@ -109,3 +109,4 @@ People
 * Johan Bergström [@jbergstroem](https://github.com/jbergstroem)
 * Ken Perkins [@kenperkins](https://github.com/kenperkins)
 * Wyatt Preul [@geek](https://github.com/geek)
+* Forrest L Norvell [@othiym23](https://github.com/othiym23)
```

Comment:
@wblankenship You might want to update this list with that one: https://github.com/orgs/nodejs/teams/build.